src/commands/ecr-login-for-secondary-account.yml

```diff
@@ -15,12 +15,19 @@ parameters:
       Name of environment variable storing the Amazon ECR repository's
       account ID
 
+  profile-name:
+    type: string
+    default: "default"
+    description: >
+      AWS profile name to be configured.
+      defaults to "default"
+
 steps:
   - run:
       name: Log into Amazon ECR
       command: |
         # aws ecr get-login returns a login command w/ a temp token
-        LOGIN_COMMAND=$(aws ecr get-login --no-include-email --region $<<parameters.region>> --registry-ids $<<parameters.account-id>>)
+        LOGIN_COMMAND=$(aws ecr get-login --no-include-email --region $<<parameters.region>> --registry-ids $<<parameters.account-id>> --profile $<<parameters.profile-name>>)
```

Comment:
I think "$" should be removed, because `profile-name` is a string parameter and not an environment variable name parameter. See: https://circleci.com/gh/CircleCI-Public/aws-ecr-orb/561 for an example of the integration test failing